src/components/views/read/cvForm/References.tsx

```diff
@@ -240,17 +355,24 @@ const References: FC<Props> = ({ nextStep, prevStep }) => {
               ) : (
                 Object.entries(card.data as ReferencesObj).map(
                   ([key, value]) => {
-                    if (
-                      value !== '' &&
-                      mapping.some((item) => item.key === key)
-                    ) {
-                      return (
-                        <Box key={key}>
-                          <Typography variant="subtitle2">
-                            {key}: {value}
-                          </Typography>
-                        </Box>
-                      );
+                    const mappingItem = mapping.find(
+                      (item) => item.key === key,
+                    );
+                    if (value !== '' && mappingItem) {
+                      const phoneNumLen =
+                        value.toString.length <= 4 &&
+                        key === 'referencePhoneNum';
```

Comment:
The phone number, since it's a text field, so the value as default country is USA for example then the value of textfield whenever i get it is (+1), which doesn't make any sense to store or show right?